packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -69,26 +69,23 @@ $ms-Persona-presenceSizeXl: 28px;
   line-height: 1;
   position: relative;
   height: $ms-Persona-sizeMd;
-  display: table;
-  table-layout: fixed;
-  border-collapse: separate;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
 
   .ms-ContextualHost {
     display: none;
   }
 }
 
 .ms-Persona-imageArea {
-  position: absolute;
+  position: relative;
```

Comment:
It seems like we need to keep this unless we set an explicit width on .ms-Persona-image